### ISSUE TITLE
On portal doesn't work

### ISSUE THREAD

**Issue by Eng1ner** [SkriptLang/Skript#4331]
### Skript/Server Version

```
Server Version: git-Paper-318 (MC: 1.16.4)
Skript Version: 2.5.3
Installed Skript Addons:
- Skellett v1.9.11
- skript-yaml v1.3.2
- Reqn v1.2.1
- SkQueryv4.1.4
```

### Bug Description

Event _on portal_ (_on player portal_ too) doesn't work both on default server world and others.

### Expected Behavior

```
on portal:
    broadcast "message"
```
After teleporting it should broadcast the message, but it doesn't work. Cancel event also doesn't work

### Steps to Reproduce

From what I remember I blocked once access to nether on server by _on portal_, but I think it was on version 1.16.1. Now it didn't work in any case I tried.

### Errors or Screenshots

There is no error both on console and in game.

### Other

I'm using on my server Multiverse NetherPortals. I have to guess, probably this cause problem, but I cannot be sure. Event didn't work both when worlds are linked and unlinked.

### Agreement

- [X] I have read the guidelines above and confirm I am following them with this report.

**Comment by AyhamAl-Ali** [comment]
This seem to work fine for me
```v
on portal:
	broadcast "1"
```
![image](https://user-images.githubusercontent.com/20037329/133341266-5866a913-1e3e-4ff1-95a7-4a197206afde.png)


The player portal event is triggered when a player gets into the portal before teleporting so it might be just not what you've expected it would do

**Comment by TPGamesNL** [comment]
Try this without Multiverse-NetherPortals. You could also try other portal events:
![image](https://user-images.githubusercontent.com/29547183/133384250-1cddca90-1e80-467d-93d4-b4fc93b6dfae.png)

You could also try changing the plugin priority in config.sk, see if that works.

**Comment by Eng1ner** [comment]
@TPGamesNL I deleted Multiverse NP, and it still doesn't working. I checked all events you send me and all of them working fine, besides _on portal_. I also changed plugin priority, without success. So I guess it's other plugin, have you idea which can it be? I don't really want to testing and deleting every plugin...
![image](https://user-images.githubusercontent.com/46759384/133394810-423aa794-b5f6-4d13-af3e-fcb9a7037912.png)
